DashAI/front/src/components/results/RunsTable.jsx

```diff
@@ -93,7 +99,10 @@ function RunsTable({ experimentId }) {
         Object.keys(run[p] ?? {}).forEach((metric) => {
           newRun = {
             ...newRun,
-            [`${getPrefix(p)}${metric}`]: run[p][metric],
+            [`${getPrefix(p)}${metric}`]:
+              typeof run[p][metric] === "number"
+                ? run[p][metric].toFixed(2)
+                : run[p][metric],
```

Comment:
This operation is a bit complex to understand. According to what I saw is: it rounds to 2 numbers if it is numeric or returns the value if it is any other type?
If so, maybe add that comment.